frontend/app/assets/javascripts/top_containers.bulk.js

```diff
@@ -539,6 +539,36 @@ function BulkActionMerge(bulkContainerSearch) {
     AS.openCustomModal("bulkActionModal", "Merge Top Containers", AS.renderTemplate("bulk_action_merge", {
       selection: self.bulkContainerSearch.get_selection()
     }), 'full');
+
+    // Access modal1 DOM
+    const $mergeBtn = $("[data-js='merge']");
+    
+    $mergeBtn.on("click", function() {
+      // Set up data for form submission
+      const victims = self.bulkContainerSearch
+                          .get_selection()
+                          .map(container => ({
+                            uri: container.uri,
+                            display_string: container.display_string
+                          }));
+
+      const target = {};
+      
+      target.el = document.querySelector('input[name="target[]"]:checked');
+      target.display_string = target.el.getAttribute('aria-label');
+      target.uri = target.el.getAttribute('value');
+
+      // Remove modal1
+      $("#bulkActionModal").remove();
+
+      // Init modal2
+      AS.openCustomModal("bulkActionModal", "Confirm Merge Top Containers", AS.renderTemplate("bulk_action_merge_confirm", {
+        victims,
+        target
+      }), 'full');
```

Comment:
Pending what @cdibella says about the content of the modal (e.g. whether it is likely to have a ton of unpredictable/user-generated text in it or not), I would suspect we'd want to remove the 'full' here so that the modal is smaller and overlays the other modal without obscuring it.

```suggestion
      }));
```

Reply:
That’s helpful, thanks for the info Lora!

Reply:
Yep, the full list of (not all that intuitively named) sizing options is here: https://github.com/archivesspace/archivesspace/blob/master/frontend/app/assets/javascripts/utils.js#L369

Reply:
@brianzelip - this looks really great! It really makes it clear what's happening in the application here.

I agree with Lora that it would be preferable if the confirmation modal overlaid the selection one rather than replaced it and if the confirmation modal were smaller than the selection one. Though what we've asked you to do is charting some new ground, that would line up more with some other confirmation-type functions in the application.

I really like how what you've done makes it very clear what's going into what. I think all the wording can stay, but the one difference that would be ideal is to remove the container that is the target of the merge from the list of what's going into it. (i.e. in the example in the video, GRAY would no longer be listed after "Merge the following top containers" ; it would only be listed after "Into this container") @lmcglohon may have already dropped you a note about that, but if not or it's not clear what I'm suggesting, just let me know.

Thanks again - so excited to see this moving along so quickly!

Reply:
Both of these TODOs are completed as of d86e001.